docs/rules/max-statements-per-line.md

```diff
@@ -0,0 +1,67 @@
+# Limit Maximum Number of Statements Per Line (max-statements-per-line)
+
+The `max-statements-per-line` rule allows you to specify the maximum number of statements allowed per line.
```

Comment:
This section is an intro to the concept that the rule tests, not about the rule itself. See http://eslint.org/docs/rules/max-nested-callbacks as an example